### ISSUE TITLE
how to preprocess raw  data?

### ISSUE THREAD

**Issue by liugaohong** [choczhang/MAPLE#2]
This work is so great,but I have a question.
It is how to preprocess the Cardiology Dataset which exist lots of nan value?

**Comment by liugaohong** [comment]
Can you public the preprocessing?

**Comment by shivin9** [comment]
@liugaohong Yes, I am also looking for the preprocessing scripts for the Sepsis prediction dataset.
I ran a test with GRASP+GRU network on another version of the sepsis dataset but cannot replicate the same performance.

My dataset includes **only** the first 24 hours of time series records and the task is to predict whether the patient will catch sepsis during his further stay (after the first 24 hours) in the ICU.

I am getting AUPRC values of 0.5 as compared to 0.7+ reported in the paper.
